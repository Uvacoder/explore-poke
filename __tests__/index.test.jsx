import { render } from "@testing-library/react";
import { MockedProvider } from '@apollo/client/testing';
import Home from "pages/index";

describe("Pokemon Home Page", () => {
  const mocks = [];

  it('should render pokemon lists component without crash', () => {
    const { container } = render(
      <MockedProvider mocks={mocks}>
        <Home />
      </MockedProvider>
    );
    expect(container).not.toBeNull();
  });

  it('should render pokemon lists', () => {
    const { container } = render(
      <MockedProvider mocks={mocks}>
        <Home />
      </MockedProvider>
    );
    expect(container).toMatchSnapshot();
  });
});