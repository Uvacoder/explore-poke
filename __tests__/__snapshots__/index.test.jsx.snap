// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pokemon Home Page should render pokemon lists 1`] = `
<div>
  <div
    class="css-7kfrid"
  >
    <div
      class="css-18muxki"
    />
  </div>
</div>
`;
